### ISSUE TITLE
Add support for additional outputs for PostgreSQL

### ISSUE THREAD

**Issue by mjf** [okbob/pspg#163]
`\d+ table`

![fr](https://user-images.githubusercontent.com/54104/114356402-d9f0a000-9b70-11eb-94d9-7e3c22ce00b4.png)

It would be really nice if the extended/additional output under the data table view was handled differently as it is not part of the tabular data (i.e. switch off numbering and handle the whole block of lines containing the `View definition:` and forth as a single row or something alike).

**Comment by okbob** [comment]
po 12. 4. 2021 v 9:30 odesílatel Matouš Jan Fialka ***@***.***>
napsal:

> \d+ table
>
> [image: fr]
> <https://user-images.githubusercontent.com/54104/114356402-d9f0a000-9b70-11eb-94d9-7e3c22ce00b4.png>
>
> It would be really nice if the extended/additional output under the data
> table view was handled differently as it is not part of the tabular data
> (i.e. switch off numbering and handle the whole block of lines containing
> the View definition: and forth as a single row or something alike).
>

It is handled differently. You can change separate colors for this area
(backgrounds, cursor, ...) - unfortunately, in this case, you need to
modify pspg source code - themes.c file

I am afraid so different counting of rows can be messy - usually one one
row is in this area.

But maybe the used theme can be fixed - usually I design minimalistic
themes with reduced colors

**Comment by okbob** [comment]
po 12. 4. 2021 v 9:50 odesílatel Pavel Stehule ***@***.***>
napsal:

>
>
> po 12. 4. 2021 v 9:30 odesílatel Matouš Jan Fialka <
> ***@***.***> napsal:
>
>> \d+ table
>>
>> [image: fr]
>> <https://user-images.githubusercontent.com/54104/114356402-d9f0a000-9b70-11eb-94d9-7e3c22ce00b4.png>
>>
>> It would be really nice if the extended/additional output under the data
>> table view was handled differently as it is not part of the tabular data
>> (i.e. switch off numbering and handle the whole block of lines containing
>> the View definition: and forth as a single row or something alike).
>>
>
> It is handled differently. You can change separate colors for this area
> (backgrounds, cursor, ...) - unfortunately, in this case, you need to
> modify pspg source code - themes.c file
>
> I am afraid so different counting of rows can be messy - usually one one
> row is in this area.
>
> But maybe the used theme can be fixed - usually I design minimalistic
> themes with reduced colors
>

And try to use border 2 - it will be more readable, I hope - with border 1
the tabular output and footer content are not well separated

Pavel

**Comment by mjf** [comment]
> And try to use border 2 - it will be more readable, I hope - with border 1
  the tabular output and footer content are not well separated

```sh
$ grep border .pspgconf 
force_uniborder = true
border_type = 2
```

**Comment by okbob** [comment]
please, check 314255491c05a77376222b5b885f28c433588570

**Comment by okbob** [comment]
fixed in 4.6.0
